### DISCUSSION TITLE
Introducing Automatic code Review

### DISCUSSION THREAD

**Discussion by NtemKenyor** [KwickerHub/frontend#1037] [General]
Hello and hope we are all doing well, Following some of the request by our contributors to always review their Pull Request on time, We have started building a system that would use two techniques: a Pipeline system and an AI system to review, comment, approve and merge pull request. 

This would make the review of projects in this organisation extremely fast.

**Comment by JasmineDavidson94** [comment]
This will be an awesome new feature! Thanks for the hard work. Looking forward to contributing more.
